### PR TITLE
Minor tweak to AI version selection

### PR DESCRIPTION
**Pull request by abays**
Just fixing some comments and using an existing variable

### CODE REVIEW

File: ansible/ocp_ai_prep.yaml [hunk 1]
```diff
@@ -589,7 +589,7 @@
           after: "assisted_service_openshift_versions_defaults:"
           before: "assisted_installer_images:"
           regexp: 'release_image:.*release_{{ ocp_version | replace(".", "\.") }}.*'
-          replace: 'release_image: "quay.io/openshift-release-dev/ocp-release:{{ ocp_version }}.{{ ocp_minor_version }}-x86_64"'
+          replace: 'release_image: "{{ ocp_release_image }}"'
```
Comment (stuggi) [inline ansible/ocp_ai_prep.yaml:592]: should do the same with url bellow to specify the url  in the conf?

Comment (abays) [reply ansible/ocp_ai_prep.yaml:592]: Which line did you have in mind?  596?

Comment (stuggi) [reply ansible/ocp_ai_prep.yaml:592]: yes, the one which has "https://amd64.ocp.releases.ci.openshift.org/releasestream/4-stable/release/{{ ocp_version }}.{{ ocp_minor_version }}"

Comment (abays) [reply ansible/ocp_ai_prep.yaml:592]: Sure, let's do that

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**